src/gui/widgets/FxLine.cpp

```diff
@@ -148,66 +175,56 @@ void FxLine::drawFxLine( QPainter* p, const FxLine *fxLine, const QString& name,
 	{
 		p->drawPixmap( 2, 0, 29, 56, *s_receiveBgArrow );
 	}
+}
 
-	// draw the channel name
-	if( m_staticTextName.text() != name )
-	{
-		// elide the name of the fxLine when its too long
-		const int maxTextHeight = 78;
-		QFontMetrics metrics( fxLine->font() );
-		QString elidedName = metrics.elidedText( name, Qt::ElideRight, maxTextHeight );
-		m_staticTextName.setText( elidedName );
-	}
-	p->rotate( -90 );
 
-	p->setFont( pointSizeF( fxLine->font(), 7.5f ) );
 
-	// Coordinates of the foreground text
-	int const textLeft = -145;
-	int const textTop = 9;
 
-	// Draw text shadow
-	p->setPen( sh_color );
-	p->drawStaticText( textLeft - 1, textTop + 1, m_staticTextName );
-	
-	// Draw foreground text
-	p->setPen( isActive ? bt_color : te_color );
-	p->drawStaticText( textLeft, textTop, m_staticTextName );
+QString FxLine::elideName( QString name )
+{
+	const int maxTextHeight = 70;
+	QFontMetrics metrics( font() );
+	QString elidedName = metrics.elidedText( name, Qt::ElideRight, maxTextHeight );
+	return elidedName;
 }
 
 
+
+
 void FxLine::paintEvent( QPaintEvent * )
 {
-	FxMixer * mix = Engine::fxMixer();
-	bool sendToThis = mix->channelSendModel(
+	bool sendToThis = Engine::fxMixer()->channelSendModel(
 		m_mv->currentFxLine()->m_channelIndex, m_channelIndex ) != NULL;
-	bool receiveFromThis = mix->channelSendModel(
+	bool receiveFromThis = Engine::fxMixer()->channelSendModel(
 		m_channelIndex, m_mv->currentFxLine()->m_channelIndex ) != NULL;
 	QPainter painter;
 	painter.begin( this );
-	drawFxLine( &painter, this,
-		mix->effectChannel( m_channelIndex )->m_name,
-		m_mv->currentFxLine() == this, sendToThis, receiveFromThis );
+	drawFxLine( &painter, this,	m_mv->currentFxLine() == this, sendToThis, receiveFromThis );
```

Comment:
If you want to keep one line, you may change the tab before `m_mv` into a space.